doc/source/whatsnew/v0.13.0.rst

```diff
@@ -153,14 +153,18 @@ API changes
 
   Added the ``.bool()`` method to ``NDFrame`` objects to facilitate evaluating of single-element boolean Series:
 
-  .. ipython:: python
+  .. code-block:: python
 
-     pd.Series([True]).bool()
-     pd.Series([False]).bool()
-     pd.DataFrame([[True]]).bool()
-     pd.DataFrame([[False]]).bool()
+     >>> pd.Series([True]).bool()
+        ..  True
+     >>> pd.Series([False]).bool()
+        ..  False
+     >>> pd.DataFrame([[True]]).bool()
+        ..  True
+     >>> pd.DataFrame([[False]]).bool()
+        ..  False
 
-- All non-Index NDFrames (``Series``, ``DataFrame``, ``Panel``, ``Panel4D``,
+- All non-Index NDFrames ( ``Series``, ``DataFrame``, ``Panel``, ``Panel4D``,
```

Comment:
I think this was unintentional? If so, can you revert.

Reply:
thanks, it was unintentional.